Automattic/wp-calypso
# Subscriptions Management: Add search to Comments tab

Closes https://github.com/Automattic/wp-calypso/issues/75848

## Proposed Changes

* Create `subscription-management/posts-search` feature flag
* Encapsulate search related code on `useSearch` hook (can be applied to Sites tab as well)
* Add search support to `usePostSubscriptionsQuery` (debounced by 300ms)
* Update Comments tab to integrate search input to query

## Testing Instructions

* Run this PR on your local env
* Go to http://calypso.localhost:3000/subscriptions/comments (with a valid subkey)
* Verify if the page rendered as expected (including search input)
* Check if the search is working as expected
* Verify if empty search results view is displayed properly
* Check if the feature flag is working as expected

## Pre-merge Checklist
